data/utils/maskFromDefects.py

```diff
@@ -79,7 +78,7 @@ def writeDefectsFile(bboxList, path):
     for bbox in bboxList:
```

Comment:
The line above looks wrong. I don't think we have `DefectListT` any more. It should say `measAlg.Defects()` and then the line 81 below could use `defectList.maskPixels()` directly.